umm-spec-lib/src/cmr/umm_spec/test/echo10_expected_conversion.clj

```diff
@@ -33,15 +33,18 @@
       (catch NumberFormatException e))))
 
 (defn- expected-file-dist-info
-  "Created expected FileDistributionInformation for ArchiveAndDistributionInformation map."
-  [file-dist-info]
-  (when-let [file-dist-info (first file-dist-info)]
-    (-> file-dist-info
-        (select-keys [:Fees :Format :FormatType])
-        (assoc :FormatType "Native")
-        (update :Fees echo10-expected-fees)
-        umm-c/map->FileDistributionInformationType
-        vector)))
+  "Created expected FileDistributionInformation for ArchiveAndDistributionInformation map.
+   In ECHO 10 there is only 1 price (fees) but there can be many data formats. The first fee
+   applies to all data formats."
+  [file-dist-infos]
+  (when file-dist-infos
+   (let [first-price (echo10-expected-fees (:Fees (first file-dist-infos)))]
```

Comment:
FileDistributionInformation is a vector of ArchiveAndDistributionInformation maps. What happens if the fee is present in the second member of the vector rather than the first? i.e.: `[{:Fees nil} {:Fees 5}]`. Or is this not a concerning use case?

Reply:
Then when we translate to ECHO 10 the price won't be translated to ECHO 10. In almost all instances the fee isn't used or None or Free is stated. There are a couple of really old collections where there is a fee, but then there is one format. While I don't think this is a problem, the possibility is there that this could occur. I could write a function that finds the first available fee and use that.

Reply:
Ok, I have added a function that finds the first FileDistributionInformation that has Fees in it.